src/plugins/intel_cpu/tests/functional/single_layer_tests/split.cpp

```diff
@@ -54,7 +54,7 @@ class SplitLayerCPUTest : public testing::WithParamInterface<splitCPUTestParams>
         int axis;
         ElementType netPrecision;
         InputShape inputShapes;
-        ov::Shape outIndices;
+        std::vector<size_t> outIndices;
```

Comment:
Just curious, what is the idea behind changing the type?

Reply:
Previously, it was InferenceEngine::SizeVector, and in most cases it was mean shape, but here `outIndices` is vector of indices, not shape. So it's more about clearness